build.gradle

```diff
@@ -0,0 +1,152 @@
+plugins {
+    id "idea"
+    id "groovy"
+    id "com.github.erdi.webdriver-binaries" version "2.2"
+    id "com.github.erdi.idea-base" version "2.2"
+    id 'com.adarshr.test-logger' version "2.0.0"
+}
+
+ext {
+    // The drivers we want to use
+    drivers = ["firefox", "chrome", "chromeHeadless", "htmlunit"]
+
+    spockVersion = "1.3-groovy-2.5"
+    gebVersion = '3.4'
+    unirestVersion = "3.7.02"
+    seleniumVersion = '3.141.59'
+    chromeDriverVersion = '83.0.4103.39'
+    geckoDriverVersion = '0.26.0'
+    htmlunitVersion = "2.36.0"
+
+    // Environment variable needed to run the tests
+    dotEnvFile = './.env'
+    odsConfigFile = '../ods-configuration/ods-core.env'
+    environmentVariables = [
+            PROV_APP_USER        : '',
+            PROV_APP_PASSWORD    : '',
+            ATLASSIAN_USER       : '',
+            ATLASSIAN_PASSWORD   : '',
+            JENKINS_USER         : '',
+            JENKINS_PASSWORD     : '',
+            OPENSHIFT_USER       : '',
+            OPENSHIFT_PASSWORD   : '',
+            PROV_APP_NAME        : '',
+            JIRA_URL             : '',
+            OPENSHIFT_PROJECT    : '',
+            OPENSHIFT_PUBLIC_HOST: '',
+            OPENSHIFT_CLUSTER    : '',
+            BITBUCKET_URL        : '',
+            SIMULATE             : ''
+    ]
+}
+
+repositories {
+    mavenCentral()
```

Comment:
Please introduce nexus! ... similar to spock-geb (https://github.com/opendevstack/ods-quickstarters/blob/master/e2e-spock-geb/files/build.gradle#L28)

Reply:
Adapated and pushed, to github, following the same way that it was the retrieval of urls/credentials done for the tests, based in ods-configuration / .env / environment variables.
Tested in an ODSBox